app/src/main/res/layout/activity_chat.xml

```diff
@@ -1,10 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
What is the difference between a constraint layout and a drawer layout? Why did you had to change this?

Reply:
DrawerLayout allows to draw interactive "drawer" views from one side of the window